.github/workflows/release.yaml

```diff
@@ -0,0 +1,85 @@
+name: release
+
+on:
+  push:
+    branches:
+      - main
+
+env:
+  IMAGE_REGISTRY: ghcr.io
+  IMAGE_REPO: ${{ github.repository_owner }}/golden-container
+  IMAGE_TAGS: latest
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    permissions:
+      # Needed for signing the container image with GitHub OIDC Token
+      id-token: write
+      # Needed to push to ghcr.io registry
+      packages: write
+
+    outputs:
+      image:  ${{ env.IMAGE_REGISTRY }}/${{ env.IMAGE_REPO }}
+      digest: ${{ steps.push-image.outputs.digest }}
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Install Cosign
+      uses: sigstore/cosign-installer@main
+
+    - name: Install Syft
+      uses: anchore/sbom-action/download-syft@v0.14.3
+
+    - name: Log in to ghcr
+      uses: redhat-actions/podman-login@v1
+      with:
+        registry: ${{ env.IMAGE_REGISTRY }}
+        username: ${{ github.actor }}
+        password: ${{ secrets.GITHUB_TOKEN }}
+
+    - name: Buildah Action
+      id: build-image
+      uses: redhat-actions/buildah-build@v2
+      with:
+        image: ${{ env.IMAGE_REPO }}
+        tags: ${{ env.IMAGE_TAGS }}
+        dockerfiles: ./Containerfile
+
+    - name: Push to ghcr
+      id: push-image
+      uses: redhat-actions/push-to-registry@v2
+      with:
+        registry: ${{ env.IMAGE_REGISTRY }}
+        image: ${{ steps.build-image.outputs.image }}
+        tags: ${{ steps.build-image.outputs.tags }}
+        username: ${{ github.actor }}
```

Comment:
The `podman-login` action earlier should make this not required, right?

Reply:
I think so too. Credentials get saved into a file, right? See https://docs.podman.io/en/latest/markdown/podman-login.1.html#options mentions , so the podman push should find them there without needing to access the username /password again.